modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+----
+
+
+. Create a `ClusterLogForwarder` CR YAML file that specifies the name of the secret. For example:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: cw <3>
+     type: cloudwatch <4>
+     cloudwatch:
+       groupBy: logType <5>
+       region: us-east-2 <6>
+     secret:
+        name: cw-secret <7>
+  pipelines:
+    - name: infra-logs <8>
+      inputRefs: <9>
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw <10>
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> Optional: Specify the option CloudWatch uses to group the logs. If you omit `groupBy`, its default value is `logType`.
+<6> Specify the AWS region.
+<7> Specify the name of the secret that contains your AWS credentials, which you created earlier.
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+.Example: How various configurations of the ClusterLogForwarder affect CloudWatch
+
+Suppose that you run an {product-title} cluster on AWS named `mycluster` and the value of its `infrastructureName` is `mycluster-7977k`.
+
+[source,terminal]
+----
+$ oc get Infrastructure/cluster -ojson | jq .status.infrastructureName
+"mycluster-7977k"
+----
+
+The "app" project is running a Pod called `busybox` that writes a message to stdout every three seconds:
+
+[source,terminal]
+----
+$ oc run busybox --image=busybox -- sh -c 'while true; do echo "My life is my message"; sleep 3; done'
+$ oc logs -f busybox
+My life is my message
+My life is my message
+My life is my message
+...
+----
+
+You can also get the UUID of the project.
+[source,terminal]
+----
+$ oc get ns/app -ojson | jq .metadata.uid
+"794e1e1a-b9f5-4958-a190-e76a9b53d7bf"
+----
+
+In your `ClusterLogForwarder` custom resource (CR), the `all-logs` pipeline forwards `infrastructure`, `audit`, and `application` logs to the CloudWatch  output, `cw`:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: cw
+     type: cloudwatch
+     cloudwatch:
+       region: us-east-2
+       groupBy: groupType
+     secret:
+        name: cw-secret
+  pipelines:
+    - name: all-logs
+      inputRefs:
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw
+----
+
+In CloudWatch, each AWS region contains three levels of objects:
```

Comment:
Here, you build off your content and connect your content very well and clearly. Clusterlogging has three groups in AWS, these groups contain their own log streams, and each stream has its own log event. 

In my earlier comment on this section, I expected the examples would build off one another like they do here :)